ivy_tests/test_ivy/test_frontends/test_jax/test_mathematical_functions.py

```diff
@@ -0,0 +1,39 @@
+# global
+import numpy as np
+from hypothesis import given, strategies as st
+
+# local
+import ivy
+import ivy_tests.test_ivy.helpers as helpers
+
+valid_float_dtypes = (ivy.float16, ivy.float32, ivy.float64)
```

Comment:
we shouldn't need any local `dtype` tuples, the helper function will skip all invalid dtypes automatically.

Reply:
Sure. this was just temporary way to exclude `bfloat16` because a helper data generating function (`helpers.array_values`) was failing due to it.  Waiting for it resolve then I can pass  `ivy....framework...valid_float_types` to hypothesisdirectly. It doesn't get skipped as it is in `valid_float` for some frameworks such as tf and jax.

Reply:
Actually applied a fix for `bfloat` mainly with [this line](https://github.com/unifyai/ivy/pull/1650/commits/5e8695b73739530c13d4f8c2f457c0eca1ab954c#diff-4149a9865e4cdfe7498bb8d64195d97c39287631b6b97027151af12011defab4R1230) for the helper function bcz bfloat16 and float32 are similar. The tests are passing with all valid float types for frameworks including bfloat16 for jax and tf.